### PR TITLE
Docs: proc doc for logging

### PR DESCRIPTION
**Pull request by stanlyDoge**
### Type of change
- Documentation

### Description
Procedual documentation module for changing log levels.

### Checklist
- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: documentation/book/proc-kafka-logging.adoc [hunk 1]
```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// assembly-cluster-operator.adoc
+
+include::common/attributes.adoc[]
+
+[id='kafka-logging-{context}']
+= Logging
+
+Logging in an important tool for observe and debugging container behaviour.
```
Comment (ppatierno) [inline documentation/book/proc-kafka-logging.adoc]: typo "is an important..."

[the remark this answers is not in the document]
Comment (ncbaratta) [reply documentation/book/proc-kafka-logging.adoc]: Replace 'in' with 'is'

File: documentation/book/proc-kafka-logging.adoc [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+There are very few differences for the common user.
+
+NOTE: Currently only the Topic Operator uses `log4j2` logging.
+
+This section describes how to change logging settings for the containers running with the Strimzi operator.
+
+.Procedure
+
+. You can set the log levels by specifying the loggers and their levels directly (inline) or by using a custom (external) config map.
+    ** Specifying log levels - Edit the logging resource that configures container you want. For example:
```
[the remark this answers is not in the document]
Comment (ncbaratta) [reply documentation/book/proc-kafka-logging.adoc]: "    ** " should be ".. "

File: documentation/book/proc-kafka-logging.adoc [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
++
+----
+  logging:
+    type: inline
+    loggers:
+      logger.name: "INFO"
+----
+    ** Using a custom (external) config map:
```
[the remark this answers is not in the document]
Comment (stanlyDoge) [reply documentation/book/proc-kafka-logging.adoc]: I had this info there before. But removed by mistake.

**Comment by dmuntima** [comment]
@scholzj , one suggestion in the module files. Instead of creating con-logging.adoc module, we can use that text as an introduction at the top of the assembly. 

- Create new assembly assembly-logging.adoc
- Create two procedures proc-logging.adoc
   One for inline logging
   One for logging from config map
- Create a reference module with the loggers
Include both the submodules into the assembly-logging, put the first paragraph, and then include this assembly into the `assembly-cluster-operator.

**Comment by scholzj** [comment]
@dmuntima Oh, I though that the assembly should contain always just includes. But if not then yes, it couldbe written there. +1 to what you outlined

File: documentation/book/assembly-kafka-logging.adoc [hunk 4]
```diff
@@ -0,0 +1,31 @@
+// This assembly is included in the following assemblies:
+//
+// master.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+:parent-context: {context}
+//:context: use
+include::common/attributes.adoc[]
```
Comment (ncbaratta) [inline documentation/book/assembly-kafka-logging.adoc]: We can't include this file here anymore per PR #733

File: documentation/book/proc-kafka-external-logging.adoc [hunk 5]
```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-logging.adoc
+
+include::common/attributes.adoc[]
```
Comment (ncbaratta) [inline documentation/book/proc-kafka-external-logging.adoc]: We can't include this file here anymore per PR #733

File: documentation/book/proc-kafka-inline-logging.adoc [hunk 6]
```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-logging.adoc
+
+include::common/attributes.adoc[]
```
Comment (ncbaratta) [inline documentation/book/proc-kafka-inline-logging.adoc]: We can't include this file here anymore per PR #733

**Comment by dmuntima** [comment]
@scholzj, the assembly must ideally contain an introduction covering what the user will accomplish by working through the modules in the assembly and sets the context for the user story the assembly is based on. Can include more than one paragraph. 

My bad if I have missed mentioning about this in the Node Scheduling assemblies peer review.

@stanlyDoge, I will review this PR after you move the content to an assembly.

File: documentation/book/ref-kafka-logging.adoc [hunk 7]
```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-logging.adoc
+
+[id='kafka-logging-{context}']
+= Loggers
+
+In the Strimzi several components are running. Each of these components has it's own loggers which values can be set.
```
Comment (dmuntima) [inline documentation/book/ref-kafka-logging.adoc]: Please change it to "{ProductName} consists of several components. Each component has its own loggers and is configurable. This section provides information about loggers of various components."

**Comment by dmuntima** [comment]
@scholzj , I think we should also document how to fetch the logs.

**Comment by scholzj** [comment]
@dmuntima With fetching you mean how to see / download them? I would leave that to the platform (OpenShift). The logs are not written to files. Instead they are collected by the Kubernetes / OpenShift and how you can access them / search them depends a lot on how the platform is configured. So I would probably not add anything.

File: documentation/book/proc-kafka-external-logging.adoc [hunk 8]
```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-logging.adoc
+
+[id='kafka-external-logging-{context}']
+= External logging
```
[the remark this answers is not in the document]
Comment (dmuntima) [reply documentation/book/proc-kafka-external-logging.adoc]: @stanlyDoge, do you think renaming the title to "Using ConfigMap for Logging" makes sense?

[the remark this answers is not in the document]
Comment (stanlyDoge) [reply documentation/book/proc-kafka-external-logging.adoc]: Yeah, probably yes :)

**Pull request merged**